core/src/conn.rs

```diff
@@ -1,6 +1,7 @@
 use crate::MongoQuery;
 use crate::{err::Result, Error};
-use bson::doc;
+use bson::{doc, UuidRepresentation};
+use mongodb::options::ConnectionString;
 use mongodb::{options::ClientOptions, sync::Client};
```

Comment:
```suggestion
use mongodb::options::ConnectionString;
use mongodb::{options::{ClientOptions, ConnectionString}, sync::Client};
```
[style-nit] Should we combine these two?